consensus/polybft/validator_set.go

```diff
@@ -96,6 +90,10 @@ func (vs validatorSet) Len() int {
 	return vs.validators.Len()
 }
 
-func (vs validatorSet) getQuorumSize() int64 {
-	return int64(math.Ceil((2 * float64(vs.totalVotingPower)) / 3))
+// getQuorumSize calculates quorum size as 2/3 super-majority of provided total voting power
+func getQuorumSize(totalVotingPower *big.Int) *big.Int {
+	// TODO: @Stefan-Ethernal math.Ceil for big.Int?
+	quorum := new(big.Int)
+
+	return quorum.Mul(totalVotingPower, big.NewInt(2)).Div(quorum, big.NewInt(3))
```

Comment:
We are lacking math.Ceil here in order to have 2/3 super-majority. Not sure how to accomplish ceil for big.int though.

Reply:
Fixed https://github.com/0xPolygon/polygon-edge/pull/1060/commits/5de8ba548d9ddc7ef5ceca003554710d449d5142